msprime/intervals.py

```diff
@@ -198,9 +308,11 @@ class RecombinationMap:
         the largest possible value, allowing the maximum resolution
         in the recombination process. However, for a finite sites
         model this can be set to smaller values.
+    :param str name: A name to use for this map, and stored in the ``.name``
```

Comment:
What's this for? This is a legacy interface, we don't want to add any more functionality to it.

Reply:
We need it to be able to convert old recombination maps (in stdpopsim) to the new style, which can then be output using the (again, new) write_hapmap function, and read-in by the new code. I discussed this with @grahamgower - perhaps you have a different suggestion for how to produce new maps from the old ones, that are (sometimes) lacking genetic position?